index.html

```diff
@@ -9,7 +9,7 @@
 </head>
```

Comment:
- the page title should adequately and briefly describe the content of the page. kindly change `My Portfolio ` with something more descriptive like :
  * *Mark's portifolio*
  * *Mark full-stack webdeveloper*


![image](https://user-images.githubusercontent.com/89282221/187498696-994aa380-626d-4a9e-ad55-ce9437c85483.png)